src/components/Pagination/Pagination.stories.tsx

```diff
@@ -32,10 +38,9 @@ const argTypes = {
     control: { type: 'range', min: 200, max: 300, step: 50 },
   },
 };
-
 export const Playground = ({ playgroundWidth, onChange, ...args }: StoryProps) => {
   // Parent component is in charge of the state management
-  const [value, setValue] = useState<string>(lotOptions[0]);
+  const [value, setValue] = useState<PaginationOptionValue>(lotOptions[0].value ?? lotOptions[0]);
```

Comment:
is this `??` really necessary? isn't this just the hardcoded string from above, which you know is defined?

Reply:
I think I could have used `lotOptions[0].value!` as well typescript was griping at me here